pkg/kustomize/kustomize_test.go

```diff
@@ -141,3 +141,29 @@ func TestExecAdd(t *testing.T) {
 		})
 	}
 }
+
+func TestCheckPresence(t *testing.T) {
+	testCases := []struct {
+		desc          string
+		expected      []string
+		input         []string
+		kustomization string
+	}{
+		{
+			desc:          "some res in kustomization.yaml",
```

Comment:
This is not the real use case, because there is the if on the patch type.

It is possible to add a test on the execAdd function?

Reply:
added